requirements.txt

```diff
@@ -0,0 +1,8 @@
+-e git+http://github.com/openaustralia/scraperwiki-python.git@morph_defaults#egg=scraperwiki
```

Comment:
Here's their new platform:

https://scraperwiki.com/datasets

In fact, I'm not sure why I have a free account at all. That landing page says that they only create new accounts for $2000/mo. Maybe I was grandfathered in or something.